parm/config/config.resources.emc.dyn

```diff
@@ -192,15 +195,16 @@ elif [ $step = "fcst" ]; then
     export npe_node_fcst=$(echo "$npe_node_max / $nth_fcst" | bc)
     export npe_node_fcst_gfs=$(echo "$npe_node_max / $nth_fcst_gfs" | bc)
     if [[ "$machine" == "WCOSS2" ]]; then
+       export wtime_fcst_gfs="02:30:00"
```

Comment:
Is this really necessary?
Or is WCOSS2 really that fast, that `6:00:00` on line 190 a "safe bet"?

Reply:
The 6hr gfs runtime covers the job for all resolutions on all machines, so yeah, line 190 is the "safe bet". It's not super necessary (beyond the possibility of jobs waiting longer on WCOSS2 down the road when the machine gets busy) so I have removed the line. Can review runtimes again later.